release-notes/next.yml

```
@@ -25,6 +25,8 @@ releases:
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1641"
       - text: "Complete composefs integration in Fedora CoreOS"
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1718"
+      - text: "nodes with cgroup v1 will not boot"
```

maybe let's add a little more context:

```suggestion
      - text: "systemd v256: nodes utilizing cgroups v1 will fail to boot"
```